we do have this function- but the autoconf doesn't detect it properly. work
around a static/non-static mixed declaration by making it match the headers
diff --git a/main/utils.c b/main/utils.c
index 6111b86..f989cca 100644
--- a/main/utils.c
+++ b/main/utils.c
@@ -96,7 +96,7 @@ AST_MUTEX_DEFINE_STATIC(__mutex);
 routine is derived from code originally written and placed in the public
 domain by Enzo Michelangeli <em@em.no-ip.com> */
 
-static int gethostbyname_r (const char *name, struct hostent *ret, char *buf,
+int gethostbyname_r (const char *name, struct hostent *ret, char *buf,
 				size_t buflen, struct hostent **result,
 				int *h_errnop)
 {
